packages/validation/src/permissions/research-output.ts

```diff
@@ -31,3 +35,62 @@ export const hasWorkingGroupsCreateUpdateResearchOutputPermissions = (
     return false;
   });
 };
+
+export const noPermissions: userPermissions = {
+  createDraft: false,
+  editDraft: false,
+  publishDraft: false,
+  editPublished: false,
+};
+
+export const isUserAsapStaff = (user: Omit<User, 'algoliaApiKey'>): boolean =>
+  user.teams.some((team) => team.role === 'ASAP Staff');
```

Comment:
I believe this is not how we want to verify if the user is staff, this function is now verifying the role inside a team

![Screenshot 2023-02-21 at 13 57 51](https://user-images.githubusercontent.com/16595804/220410619-73b57d47-b5cd-4698-b359-459c7384c74f.png)

but from the ticket
![Screenshot 2023-02-21 at 13 59 11](https://user-images.githubusercontent.com/16595804/220410888-a0b6e538-05d3-4cf4-954c-e3d40fe38967.png)

it seems that we actually want to check this another field `ASAP Hub Role` from `users` entity
![Screenshot 2023-02-21 at 13 57 04](https://user-images.githubusercontent.com/16595804/220411150-e0e59708-c5c4-458e-a4cb-3aedf482cc8c.png)